image/docker_schema1.go

```diff
@@ -112,6 +113,17 @@ func (m *manifestSchema1) ConfigBlob() ([]byte, error) {
 	return nil, nil
 }
 
+// Config returns the image configuration as per OCI v1 image-spec
+func (m *manifestSchema1) Config() (*imgspecv1.Image, error) {
```

Comment:
This _really_ needs to share code with the existing conversion code; see what `manifestSchema1.convertToManifestSchema2`/`configJSONFromV1Config` does. (I guess this can chain through the schema1→schema2, and schema2→OCI conversions, instead of writing a new schema1→OCI one..)